### ISSUE TITLE
[LINUX] not every string is on the right language

### ISSUE THREAD

**Issue by DioEgizio** [PolyMC/PolyMC#617]
### Operating System

Linux

### Version of PolyMC

PolyMC 1.2.2 (but happens on any :P)

### Description of bug

even if i have polymc in english, some stuff is in italian (my system language)

![immagine](https://user-images.githubusercontent.com/83089242/169702181-23762c36-5d37-44ab-80a5-72d86d5d8bae.png)

### Steps to reproduce

.

### Suspected cause

*shrug*

### This issue is unique

- [X] I have searched the issue tracker and did not find an issue describing my bug.

**Comment by d-513** [comment]
Qt shit?

**Comment by Scrumplex** [comment]
Probably need to tell Qt what language to use for those.

**Comment by d-513** [comment]
Yeah, it seems Qt is doing it’s own thing because it manages the buttons

**Comment by Sneedplex** [comment]
This is a non-issue. The native QDialogButtonBoxes use the system language, not the application language. There's probably a workaround, though.
